src/lang/en-US.ts

```diff
@@ -1259,6 +1260,18 @@ const FieldSelection: ResolvedIntlConfig['messages'] = {
     'fieldSelection.table.label.unknown': `Unknown`,
 };
 
+const Notifications: ResolvedIntlConfig['messages'] = {
+    'notifications.paymentMethods.missing.title': `Missing Payment Method`,
+    'notifications.paymentMethods.missing.cta': `add a payment method`,
+    'notifications.paymentMethods.missing.cta.alreadyThere': `add a payment method below`,
+    'notifications.paymentMethods.missing.trialCurrent': `The free trial for {tenant} ends in {daysLeft} days, but no payment method has been added to your account.`,
+    'notifications.paymentMethods.missing.trialCurrent.instructions': `Please {cta} before your trial ends to continue using Estuary Flow.`,
+    'notifications.paymentMethods.missing.trialEndsToday': `The free trial for {tenant} ends today, but no payment method has been added to your account`,
```

Comment:
> 'notifications.paymentMethods.missing.trialEndsToday': `The free trial for {tenant} ends today, but no payment method has been added to your account`,

A period is missing at the end of this sentence.

Reply:
fixed